app/models/get_involved.rb

```diff
@@ -5,4 +5,44 @@ def initialize(content_store_response, override_content_store_hash: nil)
     super(content_store_response, override_content_store_hash:)
     @take_part_pages = content_store_hash.dig("links", "take_part_pages")
   end
+
+  def open_consultation_count
+    GdsApi.search.search({ filter_content_store_document_type: "open_consultation", count: 0 })["total"]
+  end
+
+  def closed_consultation_count
+    query = {
+      filter_content_store_document_type: "closed_consultation",
+      filter_end_date: "from: #{1.year.ago}",
+      count: 0,
+    }
+
+    GdsApi.search.search(query)["total"]
+  end
+
+  def next_closing_consultation
+    query = {
+      filter_content_store_document_type: "open_consultation",
+      filter_end_date: "from: #{Time.zone.now.to_date}",
+      fields: "end_date,title,link",
+      order: "end_date",
+      count: 1,
+    }
+
+    GdsApi.search.search(query)["results"].first
+  end
+
+  def time_until_closure(consultation)
```

Comment:
This method has moved about a bit. Considering that's looking up specific "get_involved" translation keys it should be in the presenter.